.github/workflows/run-core-traits-tests.yml

```
@@ -11,7 +11,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
```

Pinning to the latest working version of 3.13.0.

```suggestion
        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13.0-alpha.3']
```